lib/pbench/server/api/resources/query_apis/query_controller_list.py

```diff
@@ -16,52 +21,10 @@ def __init__(self, config, logger):
         self.es_url = get_es_url(config)
         self.prefix = get_index_prefix(config)
 
-    @staticmethod
-    def _genMonthUriRange(prefix, index, start, end):
-        """
-        _genMonthUriRange Construct a comma-separated list of index names
-        qualified by year and month suitable for use in the Elasticsearch
-        /_search query URI.
-
-        The month is incremented by 1 from "start" to "end"; for example,
-
-        _genMonthUriRange('drb.', 'v4.run.', '2020-08', '2020-10') will result
-        in
-
-            'drb.v4.run.2020-08,drb.v4.run.2020-09,drb.v4.run.2020-10,'
-
-        This code is adapted from Javascript in the Pbench dashboard's
-        'moment_constants.js' getAllMonthsWithinRange function.
-
-        Args:
-            prefix ([string]): The Pbench server index prefix
-            index ([string]): The desired monthly index root
-            start ([datetime.datetime]): The start time
-            end ([datetime.datetime]): The end time
-
-        Returns:
-            [string]: A comma-separated list of month-qualified index names
-        """
-        monthResults = list()
-        queryString = ""
-        for m in rrule.rrule(rrule.MONTHLY, dtstart=start, until=end):
-            monthResults.append(m.strftime("%Y-%m"))
-
-        # TODO: hardcoding the index here is risky. We need a framework to
-        # help the web services understand index versions and template
-        # formats, probably by building a persistent database from the
-        # index template documents at startup. This is TBD.
-        for monthValue in monthResults:
-            if index == "v4.result-data.":
-                queryString += f"{prefix + index + monthValue}-*,"
-            else:
-                queryString += f"{prefix + index + monthValue},"
-        return queryString
-
     def post(self):
         """
-        POST for Pbench controller datasets with start times within the
-        specified search range, and owned by a specified user:
+        POST for Pbench controllers with datasets within the specified date
+        range, and owned by a specified user:
 
         {
             "user": "email-tag",
```

Comment:
Shouldn't this parameter be hidden now for this API?

The current use in the dashboard is to view all controllers.  So we want to query for all "public" data and all data from the current user if there is one.

Reply:
Seems like we really need clarity on the use case from the Dashboard.